README.md

```diff
@@ -3,40 +3,49 @@ Fill in the text below with your own answers to each question. Each person in yo
 
 ## About You
 What are your names, as they appear on the class roster?
-> ...
+> Kathryn Ruth Yetter maybe Kathryn Yetter, I've haven't seen the roster
+> Jonah Zimmermann
 
 What are your GitHub usernames?
-> ...
+> K: yetterka
+> Z: zimmerma2
 
 What do you prefer to be called?
-> ...
+> K: Kathryn
+> J: Jonah
 
 Why are you taking this course?
-> ...
+> K: Partially bc its part of the major requirements also because I like computing also because it fit well and sounded rad
+> J: Kathryn worded it better than I could
 
 What are your concerns about this course?
-> ...
+> K : Workload aknd being very very rusty post semester abroad
+> Z : See above
 
 What topic in this course are you most excited to cover?
-> ...
+> K: No idea, what topics are we covering??
+> J: See above
 
 Do you have any questions for me?
-> ...
+> K: How are you doing today?
+> J: "Who do you think would win in a fight, a trillion lions or the sun?" - Kathryn Yetter, 2017
```

Comment:
According to Google, the average mass of a male lion is 190.5kg. With a trillion of them, the sun would still be about  4% more massive than the lions, and larger if there were any female lions. While lions can bite things, which may or may not be useful against a star, the sun has nuclear fusion so I'd give a slight edge to the sun.